src/lib/spinel/radio_spinel_impl.hpp

```diff
@@ -158,6 +158,45 @@ static inline void LogIfFail(const char *aText, otError aError)
     }
 }
 
+template <typename InterfaceType> inline bool RadioSpinel<InterfaceType>::IsFrameForUs(spinel_iid_t aIid)
+{
+    bool found = false;
+
+    for (uint8_t i = 0; i < OT_ARRAY_LENGTH(mIidList); i++)
+    {
+        if (mIidList[i] == aIid)
+        {
+            ExitNow(found = true);
+        }
+    }
+
+exit:
+    return found;
+}
+
+template <typename InterfaceType> otError RadioSpinel<InterfaceType>::AddToIidList(spinel_iid_t aIid)
```

Comment:
Seems like `AddToIidList()` is not called from anywhere in the code/PR?

Can we remove this?

Reply:
Yes, removed it. I added this method if someone wants to add an IID during runtime, but yeah it can be added later if we need one.